### PR TITLE
feat: changes backgound-color at ::selection attribute

### PR DESCRIPTION
**Pull request by GabrielBittar**
Closes issue #2412 

## Changes proposed

Background-color for selection changed to purple, as discussed in issue #2412.

### CODE REVIEW

**Review by rupali-codes — CHANGES_REQUESTED** [review]
Hey @GabrielBittar could you please change the text color to light gray so it would look a little better.

image

**Comment by GabrielBittar** [comment]
> Hey @GabrielBittar could you please change the text color to light gray so it would look a little better.


Done

File: styles/globals.css [hunk 1]
```diff
@@ -10,6 +10,11 @@ body {
   font-family: 'Urbanist', sans-serif;
 }
 
+::selection{
+  background-color: #714eff;
+  color: lightgray;
```
Comment (rupali-codes) [inline styles/globals.css:15]:
```suggestion
  color: #e5e7eb;
```

Comment (CBID2) [reply styles/globals.css:15]: Hey @GabrielBittar, it looks like there needs to be another change here.

Comment (GabrielBittar) [reply styles/globals.css]: Sorry! I'll start using hexcodes only.

**Comment by GabrielBittar** [comment]
Done

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**